app/src/features/projects/components/ProjectFundingItemForm.tsx

```diff
@@ -99,147 +109,146 @@ const ProjectFundingItemForm: React.FC<IProjectFundingItemFormProps> = (props) =
               <DialogTitle id="alert-dialog-title">Add Funding Source</DialogTitle>
               <DialogContent>
                 <DialogContentText id="alert-dialog-description">Enter Funding Source Details</DialogContentText>
-                <Grid container spacing={3}>
-                  <Grid item xs={12}>
-                    <FormControl fullWidth variant="outlined" required={true} style={{ width: '100%' }}>
-                      <InputLabel id="agency_id-label">Agency Name</InputLabel>
-                      <Select
-                        id="agency_id"
-                        name="agency_id"
-                        labelId="agency_id-label"
-                        label="Agency Name"
-                        value={values.agency_id}
-                        onChange={(event) => {
-                          handleChange(event);
-                          // investment_action_category is dependent on agency_id, so reset it if agency_id changes
-                          setFieldValue(
-                            'investment_action_category',
-                            ProjectFundingFormArrayItemInitialValues.investment_action_category
-                          );
-
-                          // If an agency_id with a `Not Applicable` investment_action_category is chosen, auto select
-                          // it for the user.
-                          if (event.target.value !== 1 && event.target.value !== 2) {
+                <Box py={2}>
```

Comment:
Nothing should have really changed here, other than wrapping this all in this <Box>